src/components/SlateEditor/plugins/concept/block/interfaces.ts

```diff
@@ -10,7 +10,8 @@ import { ConceptEmbedData } from '@ndla/types-embed';
 import { Descendant } from 'slate';
 
 export interface ConceptBlockElement {
-  type: 'concept-block';
+  type: 'concept-block' | 'gloss-block';
```

Comment:
Kan vurdere å bruke typene som er definert i types ?

Reply:
SIden dette er standarden gjennom pluginene  fra før så kan vi ha en diskusjon på det senere. Da må vi gå for `typeof TYPE_CONCEPT_BLOCK`